addons/token-exchange/manifests/spoke_clusterrole.yaml

```diff
@@ -20,4 +20,4 @@ rules:
   verbs: ["get", "list", "watch", "create", "update", "delete"]
 - apiGroups: ["objectbucket.io"]
   resources: ["objectbucketclaims"]
-  verbs: ["get", "create",]
+  verbs: ["get", "create"]
```

Comment:
keep it in a separate PR. It's unrelated to this.

Reply:
ack